### DISCUSSION TITLE
Is there an update of Voyager coming?

### DISCUSSION THREAD

**Discussion by gcorp-llc** [thedevdojo/voyager#5806] [General]
Voyager has been a popular management panel for Laravel for a long time. We were waiting for Voyager version 2 and many changes.
But apparently it has been fixed in version 1.6
And inevitably, during this time, I used the Filament panel, and the points I saw in creating the user interface were interesting to me, such as:
Using Livewire 3
Various designs for inputs
Using sessions and cards and wizards
But the void in the management of user roles along with the management of database and site media and many tools that were inside Voyager was strongly felt.

I am hopeful that the Voyager project will continue and we will see a change in the appearance of the elements in addition to the previous facilities
I was very interested to have a role in updating Voyager, but unfortunately, I am not capable enough in programming so beautifully.
Again, I am hopeful that the Voyager team will come back with creative ideas.

**Comment by hoangnamitc** [comment]
Perhaps there are currently too many Laravel Admins on the market (nova, orchild, filament,...) causing Voyager to slow down (or stop).
I think Voyager would be great, if only it upgraded to Bootstrap v5 and added a few features
